libs/core/src/components/sage-checkbox/sage-checkbox.scss

```diff
@@ -0,0 +1,125 @@
+:host {
+  --border-color: #D3D5D9;
+  --checked-color: #040506;
+  --disabled-color: #ECEEEF;
+  --disabled-background-color: #F6F8F8;
+  --disabled-border-color: #8D939A;
+  --disabled-message-color: #B5BAC0;
+  --hover-background-color: #F9FAFA;
+  --hover-border-color: #B5BAC0;
+  --invalid-color: #CF3C32;
+  --invalid-focus-color: #FDB0AA;
+  --focus-color: #8ECAFB;
+  --message-color: #60666C;
+}
+
+.sage-checkbox {
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
+
+  &.is-invalid {
+    input {
+      border-color: var(--invalid-color);
+
+      &:checked {
+        background: var(--invalid-color);
+        border-color: var(--invalid-color);
+      }
+
+      &:focus-visible {
+        box-shadow: 0 0 0 4px var(--invalid-focus-color);
+        outline: none;
+      }
+    }
+
+    label,
+    .sage-checkbox__message {
+      color: var(--invalid-color);
+    }
+  }
+
+  &.is-indeterminate {
+    input:checked {
+      &::after {
+        border-bottom: 0;
+        transform: rotate(87deg) translate3d(-114%, 25%, 0);
+      }
+    }
+  }
+}
+
+input {
+  appearance: none;
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  height: 16px;
+  margin: 0;
+  position: relative;
+  width: 16px;
+
+  &:hover {
+    background: var(--hover-background-color);
+    border-color: var(--hover-border-color);
+  }
+
+  &:checked {
+    background: var(--checked-color);
+    border-color: var(--checked-color);
+
+    &::after {
+      border: 1px solid #FFF;
+      border-left: 0;
+      border-top: 0;
+      content: "";
+      display: block;
+      height: 7px;
+      left: 50%;
+      position: absolute;
+      top: 50%;
+      transform: rotate(43deg) translate3d(-110%, -30%, 0);
+      width: 4px;
+    }
+  }
+
+  &:disabled {
+    background: var(--disabled-background-color);
+    border-color: var(--disabled-color);
+    cursor: not-allowed;
+
+    &:checked {
+      background: var(--disabled-color);
+      border-color: var(--disabled-color);
+
+      &::after {
+        border-color: var(--disabled-border-color);
+      }
+    }
+
+    + label {
+      color: var(--hover-border-color);
+      cursor: not-allowed;
+    }
+
+    ~ .sage-checkbox__message {
+      color: var(--disabled-message-color);
+    }
+  }
+
+  &:focus-visible {
+    box-shadow: 0 0 0 4px var(--focus-color);
+    outline: none;
+  }
+}
+
+label {
```

Comment:
Might make sense to update to `input + label` - your call

Reply:
Only reason I can see is if someone passed a label into a slot. Upon further investigation I don't see a slot so probably unnecessary.

Reply:
nvm - checkboxes can't be nested, disregard.